examples/case_studies/photos/index.xml

```diff
@@ -168,7 +168,7 @@
         </view>
       </view>
       <section-list>
-        <section key="stories">
```

Comment:
shouldn't these keys preserved / added to the schema? (as they generate RN warnings when missing)

Reply:
`key` is required on `<item>` elements, but I don't think it's required on `<section>`. I checked our templates in the instawork repo and we don't use keys there.